alertmanager/router.go

```diff
@@ -0,0 +1,47 @@
+package alertmanager
+
+import (
+	"sync"
+
+	"github.com/weaveworks/mesh"
+)
+
+// Router is the interface we use for a mesh router.
```

Comment:
I'd probably also call the file gossip.go too, but then we already have mesh.go, and now I'm wondering if the two files should be combined.